llvm/test/MC/RISCV/rv32zcmop-invalid.s

```diff
@@ -1,7 +1,7 @@
-# RUN: not llvm-mc -triple riscv32 -mattr=+zcmop < %s 2>&1 | FileCheck %s
+# RUN: not llvm-mc -triple riscv32 -mattr=+zc.mop < %s 2>&1 | FileCheck %s
```

Comment:
```suggestion
# RUN: not llvm-mc -triple riscv32 -mattr=+zcmop < %s 2>&1 | FileCheck %s
```

Reply:
oops. search and replace got away from me